src/backend/access/aocs/aocsam_handler.c

```diff
@@ -2152,7 +2165,10 @@ aoco_index_build_range_scan(Relation heapRelation,
 								&& common_start_rownum <= dirEntries[colIdx].range.lastRowNum);
 
 				/* read the varblock we've just positioned to */
-				err = datumstreamread_block(aocoscan->columnScanInfo.ds[attno], NULL, attno);
+#ifdef USE_ASSERT_CHECKING
+				err =
+#endif
```

Comment:
Use one ifdef instead of two
```suggestion
#ifdef USE_ASSERT_CHECKING
				int err =
#endif
```